src/pages/CropSidebar/Filters/Filters.js

```diff
@@ -1,13 +1,15 @@
 import { Chip, Grid, Tooltip } from '@mui/material';
 import HelpOutlineIcon from '@mui/icons-material/HelpOutline';
 import React, {
-  useEffect, useContext, useState,
+  useEffect, useState,
 } from 'react';
-import { Context } from '../../../store/Store';
+import { useDispatch, useSelector } from 'react-redux';
+import { filterOffRedux, filterOnRedux, filterToggle } from '../../../reduxStore/filterSlice';
 
 const DollarsAndRatings = ({ filter, handleChange }) => {
```

Comment:
Related to the state prop definition sent to DollarsAndRatings.. it doesn't look like its being used here.